web/src/compose_validate.ts

```diff
@@ -409,14 +439,21 @@ function check_unsubscribed_stream_for_send(stream_name, autosubscribe) {
     return result;
 }
 
-function is_recipient_large_stream() {
+function is_recipient_large_stream(): boolean | number | undefined {
+    const stream_id = compose_state.stream_id();
+    if (stream_id === undefined) {
+        return undefined;
```

Comment:
This should be `false`. Note that the line just after this checks `compose_state.stream_id()` and that check should be removed.